### PR TITLE
Enhancement in Makefile : add GDB option

### PR DESCRIPTION
**Pull request by ShuffleWire**
Adding -ggdb option in makefile help (a lot) GDB to debug, which it needed because of the usage of QT, probably.

### CODE REVIEW

**Comment by omichel** [comment]
It would be probably useful to add it also to the `ode` and `wren` Makefiles.

**Comment by ShuffleWire** [comment]
I will do so as soon as I've some time.
PS : Sorry for closing/reopening the PR, i've missclicked...

**Comment by ShuffleWire** [comment]
As a side note, I'm not sure if we need this option for `ode` and `wren`, because it's initially caused by the use of Qt, and they don't depend on it, exact ? I've not tried to build and debug them, to be fair.
If needed, I'm not sure were to add it, because in both `ode` and `wren` there is no place to add debbuging options. Found one in Makefile.include, however. So I've made both modification, but only one of each should be enough.

File: src/wren/Makefile [hunk 1]
```diff
@@ -25,6 +25,10 @@ STATIC_LIBRARY = true
 CFLAGS = -std=c++11 -DGLM_ENABLE_EXPERIMENTAL
 INCLUDE = -I$(WEBOTS_HOME_PATH)/include -I$(WEBOTS_HOME_PATH)/include/glad -I../glm -I../webots/external/siphash/
 
+ifeq ($(MAKECMDGOALS),debug)
+  CFLAGS += -g -ggdb
+endif
+
```
Comment (omichel) [inline src/wren/Makefile:31]:
```suggestion
```

File: src/ode/Makefile [hunk 2]
```diff
@@ -63,6 +63,9 @@ endif
 ifeq ($(MAKECMDGOALS),release)
   CFLAGS += -DdNODEBUG
 endif
+ifeq ($(MAKECMDGOALS),debig)
+  CFLAGS += -g -ggdb
+endif
```
Comment (omichel) [inline src/ode/Makefile:68]:
```suggestion
```

**Review by omichel — CHANGES_REQUESTED** [review]
I believe it is sufficient to apply it in the Makefile.include as this one is included by the ODE and WREN makefiles.

**Comment by omichel** [comment]
Also, I believe that we should remove the `-g` option and leave only the `-ggdb` one, which is seems indeed better, see: https://stackoverflow.com/questions/668962/what-is-the-difference-between-gcc-ggdb-and-gcc-g

**Comment by ShuffleWire** [comment]
As far as I understand, the -g option is more "standard". Whould removing it could cause issue when debugging with any thing others than gdb ? If not, or if it's not a issue, I would remove it.

**Comment by ShuffleWire** [comment]
That it, finally just replacing -g with -ggdb in two location

**Review by omichel — APPROVED** [review]
Thank you.

**Pull request merged**